server/models/Profile.js

```diff
@@ -21,27 +20,38 @@ const profileSchema = new mongoose.Schema({
   },
   profilePic: {
     type: String,
+    default: "",
   },
   galleryPics: {
     type: [String],
+    default: [],
   },
   description: {
     type: String,
   },
   availability: {
-    type: [String],
+    dateRange: { startDate: String, endDate: String },
+    weeklyTimeRange: [{ startTime: String, endTime: String }],
```

Comment:
Hey @BerryGIT-ME, as we were discussed this in our meeting today, a small change maybe required here :)

Reply:
Hey @rajivtitus, @mayanksuchde, and I continued the discussion for some time after the meeting and came to an agreement that we would implement it this way.  
We think the UI would be easier on users if a user can just select a duration of days, however long or short, and then select consistent times throughout the week which can be represented by an array of seven objects representing Sundays to Saturdays. 
A typical example should be `dateRange: { startDate: '10-7-2021', endDate: '29-8-2021`} timeRange: `[{startTime: '9am', endTime: '11am'} ...{startTime: '7pm', endTime: '11pm'}]` and `timeRange[0]` maps to sundays and `timeRange[6]` maps to saturdays

Reply:
Sounds great, good work here! Thank for the clarification and example 👍🏽